lblod/app-digitaal-loket
# Bundle new forms - "Aanduiding en eedaflegging van de aangewezen burgemeester" - "Strandconcessies - reddingsdiensten kustgemeenten" - "Melding onvolledigheid inzending eredienstbestuur" - "Opstart beroepsprocedure naar aanleiding van een beslissing"

# Description

DL-5669 - DL-5625 - DL-5643 - DL-5646 - DL-5665 - DL-5670

This PR adds  : 

### Five new forms

- BesluitDocumentType - Aanduiding en eedaflegging van de aangewezen burgemeester -> <https://data.vlaanderen.be/id/concept/BesluitDocumentType/a48f2ded-78a5-45e5-a5d8-077814cec39f>

__Preview__ 
![Screenshot 2024-02-13 at 18-22-57 Loket voor lokale besturen](https://github.com/lblod/manage-submission-form-tooling/assets/38471754/e4d78579-39db-470c-8f08-13bc4a6147a4)

- BesluitDocumentType - Strandconcessies - reddingsdiensten kustgemeenten -> <https://data.vlaanderen.be/id/concept/BesluitDocumentType/ae6168d2-8119-46b1-b810-9f529be95a62> 

__Preview__ 
![Screenshot 2024-02-13 at 18-25-29 Loket voor lokale besturen](https://github.com/lblod/manage-submission-form-tooling/assets/38471754/3338ccbd-a077-47be-b266-290d83159903)

- BesluitDocumentType - Melding onvolledigheid inzending eredienstbestuur -> <https://data.vlaanderen.be/id/concept/BesluitDocumentType/863caf68-97c9-4ee0-adb5-620577ea8146> 

__Preview__ 
![Screenshot 2024-02-21 at 11-23-22 Loket voor lokale besturen](https://github.com/lblod/app-digitaal-loket/assets/38471754/2fffa7f2-e8de-44d3-8680-11f03c8dac5a)

- BesluitDocumentType - Opstart beroepsprocedure naar aanleiding van een beslissing ->
<https://data.vlaanderen.be/id/concept/BesluitDocumentType/802a7e56-54f8-488d-b489-4816321fb9ae>

__Preview__ 
![Screenshot 2024-02-21 at 11-24-09 Loket voor lokale besturen](https://github.com/lblod/app-digitaal-loket/assets/38471754/eea5ccc9-c9a4-4a76-bcbe-593f0ef0f158)

- BesluitDocumentType - Afschrift erkenningszoekende besturen -> <https://data.vlaanderen.be/id/concept/BesluitDocumentType/a970c99d-c06c-4942-9815-153bf3e87df2> 

__Preview__
![Screenshot 2024-02-27 at 12-13-05 Loket voor lokale besturen](https://github.com/lblod/app-digitaal-loket/assets/38471754/820042ba-222e-4e86-8f2e-e78b02a98845)

### Adjusting alert components (alert and info) informational text in the following forms

More details in PR : https://github.com/lblod/manage-submission-form-tooling/pull/45

### Enabling Kalliope flow

Only these forms are affected : 
- Aanduiding en eedaflegging van de aangewezen burgemeester
- Strandconcessies - reddingsdiensten kustgemeenten
- Opstart beroepsprocedure naar aanleiding van een beslissing 
- Afschrift erkenningszoekende besturen

### Removing from Toezicht-ABB app export

- Melding onvolledigheid inzending eredienstbestuur 

### Adding two new codelists

`http://lblod.data.gift/concept-schemes/ba36f197-1a96-4ea2-a7f7-3b5c7ffcd6ee` form Opstart beroepsprocedure naar aanleiding van een beslissing

This codelist displays 3 options from the Type beroepsprocedure field :  Meerjarenplan(wijziging)”, “Afsprakennota” and “Budget(wijziging)”

`http://lblod.data.gift/concept-schemes/27f40c36-141d-42e9-bed8-fea1a47c4869` form "Afschrift erkenningszoekende besturen".

This codelist is giving five options to choose from the field "type afschrift" : “Begroting en jaarrekening van de lokale geloofsgemeenschap ”, “Budget”, “Jaarrekening bestuur in wachtperiode”, “Verslag van de vergaderingen van het voorlopig bestuursorgaan”, “Ontwerp van het meerjarenplan”

# Type of change


# Related services

- enrich-submission
- mu-migrations

# How to test 

:warning:  Before running the branch I suggest creating a backup as codelist is subject to change

1. Generate dist file and copy the forms to consuming apps
2. Make sure the following migration has run in Loket 
3. Log as Gemeente (All forms are visible by Gemeenten but you can chose other units based on the visibleBy rule) and Bestuur van de Eredienst for the form "Afschrift erkenningszoekende besturen"
4. Save the form, it should be saved without errors.
5. Send the form, it should be consumed by app-toezicht-abb ( except Melding onvolledigheid inzending eredienstbestuur)/ app-worship-decisions-database (Melding onvolledigheid inzending eredienstbestuur and Opstart beroepsprocedure naar aanleiding van een beslissing) and not by app-public-decisions-database.
a. To consume the form you can follow the docs @ https://github.com/lblod/app-toezicht-abb#trigger-import-export-flow-from-app-digitaal-loket
### Consuming on app-worship-decisions-database
6. Opstart beroepsprocedure naar aanleiding van een beslissing is visible by ABB and a sender selected form that list (Gemeente, Provincie, Centraal bestuur van de eredienst,  Bestuur van de eredienst or Representatief Orgaan)
7. Melding onvolledigheid inzending eredienstbestuur is visible by ABB, The selected worship service/central worship service in the form and a sender selected from that list ((Gemeente or Provincie)
8. Afschrift erkenningszoekende besturen is visible by ABB, the representatief Orgaan, Gemeenten and/or Provincies that have a local involvement and the sender Bestuur van de eredienst
 
# What to check

- Typos, and form structure  

# Links to other PR's

- Manage-submissions-form-tooling -> https://github.com/lblod/manage-submission-form-tooling/pull/44 & https://github.com/lblod/manage-submission-form-tooling/pull/45 & https://github.com/lblod/manage-submission-form-tooling/pull/46
- App-meldingsplichtige -> https://github.com/lblod/app-meldingsplichtige-api/pull/41
- App-toezicht-ABB -> https://github.com/lblod/app-toezicht-abb/pull/38
- App-public-decisions-database -> https://github.com/lblod/app-public-decisions-database/pull/25
- App-worship-decisions-database -> https://github.com/lblod/app-worship-decisions-database/pull/59
- Prepare-submissions-for-export-service -> https://github.com/lblod/prepare-submissions-for-export-service/pull/12 & https://github.com/lblod/prepare-submissions-for-export-service/pull/13
- Worship-submissions-graph-dispatcher-service -> https://github.com/lblod/worship-submissions-graph-dispatcher-service/pull/18 & https://github.com/lblod/worship-submissions-graph-dispatcher-service/pull/19
- Enrich-submission-service -> https://github.com/lblod/enrich-submission-service/pull/18 & https://github.com/lblod/enrich-submission-service/pull/19

# Notes

- drc up -d enrich-submission prepare-submissions-for-export; drc restart migrations resource cache enrich-submission
